numpy/core/tests/examples/cython/meson.build

```diff
@@ -0,0 +1,27 @@
+project('checks', 'c', 'cython')
+
+py_mod = import('python')
+py3 = py_mod.find_installation(pure: false)
```

Comment:
minor: we've started to write this as a one-liner elsewhere - a bit better practice since `py_mod` isn't needed for anything:
```meson
py3 = import('python').find_installation(pure: false)
```

For an even more minor one: we may also replace `py3` by `py`, since 2.x is well and truly gone by now.